static/app/views/replays/detail/network/details/getOutputType.tsx

```diff
@@ -16,41 +17,42 @@ type Args = {
 };
 
 export default function getOutputType({isSetup, item, visibleTab}: Args): Output {
-  const isSupportedOp = ['resource.fetch', 'resource.xhr'].includes(item.op);
-  if (!isSupportedOp) {
+  if (!isRequestFrame(item)) {
     return Output.UNSUPPORTED;
   }
 
   if (!isSetup) {
     return Output.SETUP;
   }
 
-  const request = item.data?.request ?? {};
-  const response = item.data?.response ?? {};
+  const request = item.data.request;
+  const response = item.data.response;
 
   const hasHeaders =
-    Object.keys(request.headers || {}).length ||
-    Object.keys(response.headers || {}).length;
+    Object.keys(request?.headers ?? {}).length ||
+    Object.keys(response?.headers ?? {}).length;
   if (hasHeaders && visibleTab === 'details') {
     return Output.DATA;
   }
 
-  const hasBody = request.body || response.body;
+  const hasBody = request?.body || response?.body;
   if (hasBody && ['request', 'response'].includes(visibleTab)) {
     return Output.DATA;
   }
 
-  const reqWarnings = request._meta?.warnings ?? ['URL_SKIPPED'];
-  const respWarnings = response._meta?.warnings ?? ['URL_SKIPPED'];
+  const reqWarnings = request?._meta?.warnings ?? ['URL_SKIPPED'];
+  const respWarnings = response?._meta?.warnings ?? ['URL_SKIPPED'];
   const isReqUrlSkipped = reqWarnings?.includes('URL_SKIPPED');
   const isRespUrlSkipped = respWarnings?.includes('URL_SKIPPED');
   if (isReqUrlSkipped || isRespUrlSkipped) {
     return Output.URL_SKIPPED;
   }
 
   if (['request', 'response'].includes(visibleTab)) {
-    const isReqBodySkipped = reqWarnings?.includes('BODY_SKIPPED');
-    const isRespBodySkipped = respWarnings?.includes('BODY_SKIPPED');
+    // @ts-expect-error: is BODY_SKIPPED really emitted from the SDK?
+    const isReqBodySkipped = reqWarnings.includes('BODY_SKIPPED');
+    // @ts-expect-error: is BODY_SKIPPED really emitted from the SDK?
```

Comment:
Hmm yeah I don't see this in the SDK

Reply:
To followup: https://github.com/orgs/getsentry/projects/23/views/3?pane=issue&itemId=34084925